models/issues/comment_list.go

```diff
@@ -428,6 +428,9 @@ func (comments CommentList) loadReviews(ctx context.Context) error {
 	}
 
 	for _, comment := range comments {
+		if comment.Type == CommentTypeReviewRequest {
```

Comment:
I don't understand why we shouldn't assign `Review` to this type comments which have been loaded.

Reply:
- because we just mostly get a lot of errors -> just see gitea instances who do have reviews
- because we dont use them in the comment_list template anyway (so why add another db query)

Reply:
If the comment could be written into code, and tell reader that "if the CommentTypeReviewRequest is not skipped, what's wrong would happen", then it could LGTM.

Reply:
> because we just mostly get a lot of errors -> just see gitea instances who do have reviews

I think it's better to do this below:

```
if comment.ReviewID > 0 && comment.Type != CommentTypeReviewRequest {
	log.Error(...)
}
```

Because that's the real code block which handles the "non-existing review", it just needs to "skip" the error logs, doesn't need to add a new code block.

Reply:
well there is no such code where we do `log.Error(...)`  ... in the normal case an error gets propagated back the stack - and we catch it somewhere and log it away ...

anyway I added your suggestion to where it most made sense

Reply:
(https://github.com/go-gitea/gitea/pull/28551/commits/34ea361e66e3ec963b3e29fdd3739b916a4c8461)

Reply:
ah sorry probably found the place you want to change ... well we now have a safeguard more ...